pkg/sql/instrumentation.go

```diff
@@ -470,20 +470,20 @@ func (ih *instrumentationHelper) Setup(
 		}
 	}
 
-	ih.collectExecStats = collectTxnExecStats
-
-	// Don't collect it if Stats Collection is disabled. If it is disabled the
-	// stats are not stored, so it always returns false for previouslySampled.
-	// TODO(117690): Unify StmtStatsEnable and TxnStatsEnable into a single cluster setting.
```

Comment:
Should this TODO get deleted?

Reply:
Oops, I'll put this back. It still looks relevant.